src/test/java/core/basesyntax/strategy/handler/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.strategy.handler.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.strategy.handler.OperationHandler;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private OperationHandler handler;
+    private int oldValue;
```

Comment:
let's remove this oldValue and simply use literal where needed. Fix the same issue in all places